client/src/components/SiteForm.js

```diff
@@ -0,0 +1,162 @@
+import React, { useState } from "react";
+import translateServerErrors from "../services/translateServerErrors.js";
+import { Redirect } from "react-router-dom";
+import ErrorList from "./layout/ErrorList";
+import Dropzone from "react-dropzone";
+
+const SiteForm = (props) => {
+    const { user } = props;
+    const [siteRecord, setSiteRecord] = useState({
+        name: "",
+        creatorId: user.id,
+        address: "",
+        description: "",
+        setting: "",
+        minimumAge: 0,
+        image: {},
```

Comment:
this is fine, but in future feature development adding image uploading is a separate feature as it requires an integration. We should first build out the form for all other regular fields, then once that is working tie in functionality to upload images.

You don't need to remove the functionality from this feature, but something to consider for the future